### PR TITLE
Apply topology.istio.io/network label to istio-system for MN topology

### PR DESCRIPTION
**Pull request by backward-compatible**

### CODE REVIEW

File: asmcli/commands/install.sh [hunk 1]
```diff
@@ -192,6 +192,7 @@ install_control_plane() {
   if [[ "${MANAGED}" -eq 1 ]]; then
     install_managed_control_plane
   else
+    label_istio_namespace
```
Comment (backward-compatible) [inline asmcli/commands/install.sh:195]: @zerobfd should this call be outside the surrounding if/else ? (In order to apply it for both in-cluster and MCP.)

Comment (zerobfd) [reply asmcli/commands/install.sh:195]: yep

File: asmcli/lib/context.sh [hunk 2]
```diff
@@ -45,7 +46,6 @@ context_init() {
     "USE_HUB_WIP": ${USE_HUB_WIP:-1},
     "USE_VM": ${USE_VM:-0},
     "USE_MCP_CNI": ${USE_MCP_CNI:-0},
-    "FLEET_ID": "${FLEET_ID:-}",
```
Comment (zerobfd) [inline asmcli/lib/context.sh:48]: was this removed on purpose?

Comment (backward-compatible) [reply asmcli/lib/context.sh:48]: yea that line is duplicated above..

File: asmcli/commands/install.sh (hunk 1 above)
Comment (backward-compatible) [reply asmcli/commands/install.sh]: done

**Review by zerobfd — APPROVED** [review]

**Pull request merged**